src/transformers/training_args.py

```diff
@@ -220,6 +220,9 @@ class TrainingArguments:
         sharded_ddp (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Use Sharded DDP training from `FairScale <https://github.com/facebookresearch/fairscale>`__ (in distributed
             training only). This is an experimental feature.
+        deepspeed (:obj:`str`, `optional`):
```

Comment:
Could we name this argument `deepspeed_config` without too much trouble? I forgot where we were at on this.

Reply:
we could, but it's a very meh name, IMO

As discussed originally, now this flag has a dual function - activate and config, so having just "_config" in it is mis-representative of its mnemonics.

The other option is to just leave the same as deepspeed does it:  `--deepspeed --deepspeed_config file`

I'm open to suggestions.

Reply:
`deepspeed_config_file` is a bit long but more explicit. I have no strong opinion on this, so do as you prefer.

Reply:
I asked the other devs - perhaps someone will have a better idea than ours.